catalogue/webapp/pages/concept.tsx

```diff
@@ -128,81 +109,75 @@ export const ConceptPage: NextPage<Props> = ({
   const hasImagesTabs = !!(imagesBy?.totalResults && imagesAbout?.totalResults);
 
   return (
-    // TODO fill meta information; who decides this?
+    // TODO fill meta information; wait for confirmation on description
     <CataloguePageLayout
       title={conceptResponse.label}
-      description="<TBC>"
+      description={`Find out more about ${conceptResponse.label} by browsing related works and images from Wellcome Collection.`}
       url={{ pathname: `/concepts/${conceptResponse.id}`, query: {} }}
       openGraphType="website"
       siteSection="collections"
       jsonLd={{ '@type': 'WebPage' }}
       hideNewsletterPromo={true}
     >
-      <ConceptHero
-        v={{ size: 'xl', properties: ['padding-top', 'padding-bottom'] }}
-      >
+      <ConceptHero>
         <div className="container">
-          <ConceptDescription>
-            <h1 className="font-intb">{conceptResponse.label}</h1>
-            {/* TODO dynamise */}
-            {FAKE_DATA.description && (
-              <p className="font-size-5">{FAKE_DATA.description}</p>
-            )}
-            {/* TODO dynamise */}
-            {FAKE_DATA.urls?.length > 0 &&
-              FAKE_DATA.urls.map(link => {
-                /* TODO Could they be internal links? Check if external to display arrow, decide on rel. */
-                return (
-                  <a
-                    key={link.url}
-                    href={link.url}
-                    rel="nofollow"
-                    className={font('intr', 6)}
-                  >
-                    {link.label} ↗
-                  </a>
-                );
-              })}
-          </ConceptDescription>
+          <TypeLabel>{conceptResponse.type}</TypeLabel>
+          <Space v={{ size: 's', properties: ['margin-top', 'margin-bottom'] }}>
+            {/* TODO only getting Subjects, confirm if it will change or not */}
+            <HeroTitle>{conceptResponse.label}</HeroTitle>
+            {/* TODO get copy from Jonathan */}
+            <ConceptDescription className="font-size-5">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam
+              dapibus suscipit enim nec aliquam.
+            </ConceptDescription>
+          </Space>
         </div>
       </ConceptHero>
 
+      {/* Images */}
       {hasImages && (
-        <ConceptImages
-          as="section"
-          v={{ size: 'xl', properties: ['padding-top', 'padding-bottom'] }}
-        >
+        <ConceptImages as="section">
           <div className="container">
-            <h2 className={`sectionTitle ${font('wb', 3)}`}>Images</h2>
+            <h2 className="h2 sectionTitle">Images</h2>
```

Comment:
…although I agree with this change which I guess directly contradicts the logic I gave above! I think I'd still prefer the `font` function above, but I guess my case for it isn't as strong as it once was.